exec/java-exec/src/test/java/org/apache/drill/test/QueryRowSetIterator.java

```diff
@@ -99,6 +101,13 @@ public DirectRowSet next() {
     }
   }
 
+  public void printAll() {
```

Comment:
No sure we should allow printing into stout in the code. When code was for tests, it was ok but since we moved it, I think we should consider passing the stream instead.